### PR TITLE
Add header and footer

### PR DESCRIPTION
**Pull request by codegracie**
Improve Footer with links to en.yml

Added new routes

Added images

Adding last batch of files

### CODE REVIEW

File: app/views/categories/index.html.erb [hunk 1]
```diff
@@ -3,5 +3,9 @@
     <%= t ".panel_title".html_safe %>
   </div>
   <div class="panel-body">
+
```
Comment (dieunb) [inline app/views/categories/index.html.erb]: remove any white line

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -17,9 +17,33 @@
...
+      </div> </div> </div>
+      
+      <div class="col-sm-6 col-md-4"> <div class="thumbnail"> 
+        <%= link_to image_tag("logo.png", alt: "Lessons logo", size:"400x400" ) %>
+      <div class="caption"> 
+      <p>
+        <%= link_to "#{t '.button_lessons'}".html_safe, categories_path, class: "btn btn-primary btn-lg" %>
+      </p> 
+      </div> </div> </div>
+
```
Comment (dieunb) [inline app/views/home/index.html.erb]: many white line not use

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -17,9 +17,33 @@
...
       <div class="col-md-8">
         <div class="row">
-          <%= link_to "#{t '.button_words'}".html_safe, words_path, class: "btn btn-primary btn-lg" %>
-          <%= link_to "#{t '.button_lessons'}".html_safe, categories_path, class: "btn btn-primary btn-lg" %>
+
+
+    
+        
+      <div class="col-sm-6 col-md-4"> <div class="thumbnail"> 
+          <%= link_to image_tag("logo.png", alt: "Words logo", size:"400x400" ) %>
```
Comment (dieunb) [inline app/views/home/index.html.erb]: indent (2 spaces)

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -17,9 +17,33 @@
...
+      <p>
+      <%= link_to "#{t '.button_words'}".html_safe, words_path, class: "btn btn-primary btn-lg" %>
+      </p> 
+      </div> </div> </div>
+      
+      <div class="col-sm-6 col-md-4"> <div class="thumbnail"> 
+        <%= link_to image_tag("logo.png", alt: "Lessons logo", size:"400x400" ) %>
+      <div class="caption"> 
+      <p>
+        <%= link_to "#{t '.button_lessons'}".html_safe, categories_path, class: "btn btn-primary btn-lg" %>
```
Comment (dieunb) [inline app/views/home/index.html.erb]: `<%= link_to t( ".button_lessons"}, categories_path, class: "btn btn-primary btn-lg" %>`

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -0,0 +1,16 @@
+
+<!-- Navbar -->
+<header class="navbar navbar-fixed-top navbar-inverse">
+	<div class ="container">
```
Comment (dieunb) [inline app/views/layouts/_header.html.erb]: indent

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,14 +1,19 @@
 <!DOCTYPE html>
+
 <html>
 <head>
-  <title><%= t ".title".html_safe %></title>
-  <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track": true %>
-  <%= javascript_include_tag "application", "data-turbolinks-track": true %>
+  <title><%= t '.title'.html_safe %></title>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: double quote
```
t '.title'
```

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,14 +1,19 @@
...
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: double quote

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,14 +1,19 @@
...
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: double quote

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -1,14 +1,19 @@
...
   <%= csrf_meta_tags %>
 </head>
 <body>
+   <%= render 'layouts/header' %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: double quote

File: app/views/layouts/application.html.erb [hunk 10]
```diff
@@ -1,14 +1,19 @@
...
+
   <div class="container upper-margin">
     <%= yield %>
+  	<%= render 'layouts/footer' %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: double quote

File: app/views/home/signup.html.erb [hunk 11]
```diff
@@ -0,0 +1,16 @@
+
+<div class ="page-wrapper">
+
+<div class="center jumbotron">
+  <h1>Welcome to Framgia eLearning System</h1>
```
Comment (MuhammadTamzid) [inline app/views/home/signup.html.erb]: i18n

File: app/views/home/signup.html.erb [hunk 12]
```diff
@@ -0,0 +1,16 @@
...
+
+  <h2>
+   Free online learning, and a worldwide community of native-speaking language learners just like you.
```
Comment (MuhammadTamzid) [inline app/views/home/signup.html.erb]: i18n

File: app/views/home/signup.html.erb (hunk 11 above)
Comment (codegracie) [inline app/views/home/signup.html.erb]: Added in en.yml

File: app/views/home/signup.html.erb (hunk 12 above)
Comment (codegracie) [inline app/views/home/signup.html.erb]: Added in en.yml

File: app/views/layouts/application.html.erb (hunk 10 above)
Comment (codegracie) [inline app/views/layouts/application.html.erb]: Replaced single quotes and indented

File: app/views/home/index.html.erb (hunk 4 above)
Comment (codegracie) [inline app/views/home/index.html.erb]: Modified.

File: app/views/home/index.html.erb (hunk 3 above)
Comment (codegracie) [inline app/views/home/index.html.erb]: Done. Set in sublime text too.

File: app/views/home/index.html.erb (hunk 2 above)
Comment (codegracie) [inline app/views/home/index.html.erb]: Ok, noted. done clean up.

**Pull request closed without merging**